src/registrar/forms/application_wizard.py

```diff
@@ -188,13 +207,42 @@ def from_database(self, obj):
         if alternative_domain is not None:
             self.initial["alternative_domain"] = alternative_domain.sld
 
-    requested_domain = forms.CharField(label="What .gov domain do you want?")
+    requested_domain = forms.CharField(
+        label="What .gov domain do you want?",
+    )
     alternative_domain = forms.CharField(
         required=False,
         label="Are there other domains you’d like if we can’t give you your first "
         "choice? Entering alternative domains is optional.",
     )
 
+    def clean_requested_domain(self):
+        """Requested domains need to be legal top-level domains, not subdomains.
+
+        If they end with `.gov`, then we can reasonably take that off. If they have
+        any other dots in them, raise an error.
+        """
+        requested = self.cleaned_data["requested_domain"]
+        if not requested:
+            # none or empty string
+            raise forms.ValidationError(
+                "Please enter the .gov domain that you are requesting.", code="invalid"
+            )
+        if requested.endswith(".gov"):
+            requested = requested[:-4]
+        if "." in requested:
+            raise forms.ValidationError(
+                "Please enter a top-level domain name without any periods.",
```

Comment:
This can a be a todo for later, if Michelle is going to do some content work on the error messages, but I think `top-level` might not be quite clear here

Reply:
True. .gov is the top-level domain, everything else is just a "domain" (or a second-level domain, but we don't need to use the jargon).

Reply:
Changed to just say "Please enter a domain without any periods."